assets/model_monitoring/components/model_monitor/model_monitor_output_signal_metrics/spec.yaml

```diff
@@ -0,0 +1,52 @@
+$schema: http://azureml/sdk-2-0/SparkComponent.json
+type: spark
+
+name: model_monitor_output_signal_metrics
+display_name: Model Monitor - Output Signal Metrics
+description: Output the computed model monitor metrics to the default datastore.
+version: 0.3.2
+is_deterministic: true
+
+code: ../../src/
+entry:
+  file: ./model_monitor_output_signal_metrics/run.py
+
+inputs:
+  signal_metrics: 
+    type: mltable
+    mode: direct
+  signal_name:
+    type: string
+  signal_type: 
+    type: string
+  version: 
```

Comment:
what's the version in this case?

Reply:
If this is the version for the output, we don't need to expose it as an input, we can simply keep it as a constant within the source file

Reply:
yes, so the output file will always have the same version, like `1.0.0"`. Is that right? I am unsure what this version is for.

Reply:
Updated assuming it is the output schema version (aka, `METADATA_VERSION`).

Reply:
That version is for a manual version bump when we want to do UX migration - we'll probably be using it when we migrate to outputv2 as a whole (aka from 1 to 2 or something)

Reply:
We'll be bumping very very rarely (hopefully!!)

Reply:
Sounds good! Thanks.